lib/parsers/fdx_parser.rb

```diff
@@ -24,9 +24,9 @@ def xml_check(io)
   def check_for_document_type(file_and_document_type)
     sanitized_data = file_and_document_type.downcase
     if sanitized_data.include?('finaldraft') && sanitized_data.include?('script')
-      return :fdx, :script
```

Comment:
I'd keep the `return`. It might be the "ruby way" to omit it or better, not needing it, but why giving a slight chance to make things unclear (especially to people new to Ruby) ? One more word costs us nothing and gives more clarity imho :)

Reply:
Same in another couple of references in this PR

Reply:
Don't blame me, I agree with you, blame the code style check 🤷🏼‍♂️

Reply:
really? the linter was complaining about it?

Reply:
As an alternative, I can configure the code style to not check for stuff like this, or any of the other changes that I had to make to appease it. I don't feel strongly either way.

Reply:
nah it's fine like this.. thanks anyway!